packages/react/README.md

```diff
@@ -35,37 +38,46 @@ initializeFaro({
     new ReactIntegration({
       // Only needed if you want to use the React Router instrumentation
       router: {
-        version: ReactRouterVersion.V6,
+        version: ReactRouterVersion.V6_data_router,
         dependencies: {
-          createRoutesFromChildren,
           matchRoutes,
-          Routes,
-          useLocation,
-          useNavigationType,
-        },
-      },
-
-      // Or if you use react-router v4/v5
-      router2: {
-        version: ReactRouterVersion.V5, // or ReactRouterVersion.V4,
-        dependencies: {
-          history, // the history object used by react-router
-          Route, // Route component imported from react-router package
         },
       },
     }),
   ],
 });
 ```
 
-### Use with React Router with Data Routers
+To instrument the router wrap it with the `withFaroRouterInstrumentation(dataRouter)` function,
+which attaches teh Faro instrumentations to the router.
+This is usually done in the file where you create the router, often this App.\* file.
+
+Steps:
+
+1. Create a data router (createBrowserRouter, createHashRouter, createMemoryRouter)
+2. Instrument the data router to receive route changes by wrapping it with `withFaroRouterInstrumentation()`
 
 ```ts
-import { matchRoutes } from 'react-router-dom';
+const reactBrowserRouter = createBrowserRouter([
+  //...
+]);
 
-import { getWebInstrumentations, initializeFaro, ReactIntegration, ReactRouterVersion } from '@grafana/faro-react';
-import { TracingInstrumentation } from '@grafana/faro-web-tracing';
+const browserRouter = withFaroRouterInstrumentation(reactBrowserRouter);
+```
+
+_upgrading from v6 to v6 data router_
```

Comment:
```suggestion
_upgrading from v6 without a data router to v6 with a data router_
```

Reply:
Q: What would be the best format for the description?
Heading, or bold or keep it italic or simply no formatting at all?